### PR TITLE
Italy blog post

### PR DESCRIPTION
**Pull request by drewda**

### CODE REVIEW

File: _posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+category: news
+published: false
+isThereTitle: true
+title: Expanding our collection of transit data in Italy
+---
+
+!["a train in Torre Pellice in the Piedmont region of Italy, photo by Antonio Martinetti"](https://c2.staticflickr.com/2/1510/24314365009_252b45739e_z.jpg)
+<p class='caption'>We're working to include all of the transit data feeds for Italy, and the world. Photo of a train in Torre Pellice in the Piedmont region of Italy by <a href='https://www.flickr.com/photos/tugnaz/24314365009/' target='_blank'>Antonio Martinetti</a>.</p>
+
+Since March, when we added our [first feeds of transit operators in Italy](https://transit.land/news/2016/03/24/transitland-in-italy.html) to the Transitland Feed Registry, we've added 27 more!
```
Comment (drewda) [inline _posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md]: Best to leave the domain off the URL.

File: _posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+category: news
+published: false
+isThereTitle: true
+title: Expanding our collection of transit data in Italy
+---
+
+!["a train in Torre Pellice in the Piedmont region of Italy, photo by Antonio Martinetti"](https://c2.staticflickr.com/2/1510/24314365009_252b45739e_z.jpg)
```
Comment (drewda) [inline _posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md]: If the license allows, please copy the image file into this repo so we can reference a local copy.

File: _posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+In the process I learned a little Italian, realized that "regions" are official Italian administrative boundaries ([there are 20 of them](https://en.wikipedia.org/wiki/Regions_of_Italy)), and delved into some of the complexity of European public transport association hierarchies. 
+
+Transitland's Feed Registry now has feeds from transit operators in Florence, Venice, Palermo, Naples, Milan, Bari, Turin, Rome, and Bologna, among others.  Even after processing over two dozen feeds, there are nearly 100 more left to review. And that's just from the collection that has been submitted by our generous contributors. [Check out all the Italian transit feeds](https://transit.land/feed-registry/?country=Italy).
+
+Right now I'm working through a massive combined feed for the [_Piemonte_ (Piedmont) region](https://whosonfirst.mapzen.com/spelunker/id/404227493/#7/45.275/7.920). The capital is Turin, which is also a province. The region contains part of the Alps mountain range and borders France and Switzerland.
```
Comment (drewda) [inline _posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md]: How about including a screenshot from the WoF Spelunker?

File: _posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Transitland's Feed Registry now has feeds from transit operators in Florence, Venice, Palermo, Naples, Milan, Bari, Turin, Rome, and Bologna, among others.  Even after processing over two dozen feeds, there are nearly 100 more left to review. And that's just from the collection that has been submitted by our generous contributors. [Check out all the Italian transit feeds](https://transit.land/feed-registry/?country=Italy).
+
+Right now I'm working through a massive combined feed for the [_Piemonte_ (Piedmont) region](https://whosonfirst.mapzen.com/spelunker/id/404227493/#7/45.275/7.920). The capital is Turin, which is also a province. The region contains part of the Alps mountain range and borders France and Switzerland. 
+
+The feed is managed by the Piedmont government, and comes from [their open data website](http://www.dati.piemonte.it). It contains feeds for 86 operators. Since we prefer to keep combined feeds together, this one will take longer because each of the operators' medata needs to be individually verified.
```
Comment (drewda) [inline _posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md]: "operator's metadata"?

File: _posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+category: news
+published: false
```
Comment (drewda) [inline _posts/2016-07-19-expanding-our-collection-of-transit-data-in-italy.md]: if you toggle published to true, you'll be able to view the draft at https://precog.mapzen.com/transitland/www-transit-land/italy-blog-post/news/2016/07/19/expanding-our-collection-of-transit-data-in-italy

**Comment by stevevance** [comment]
I've updated the post to address @drewda's comments

**Comment by drewda** [comment]
@stevevance a few final comments:

- please change to "Hi, I'm Steven Vance, and I started working as Transitland's data editor last week"
- there's a markdown issue in the final paragraph. An extra space?
- John ( @burritojustice ) has ideas about how to embed GeoJSON maps in the blog post. Definitely worth doing, but alright to save that for a future week's update, if it's going to take too long this time around. He's got some examples to share of how he's done this easily on mapzen.com blog posts.

**Comment by stevevance** [comment]
@drewda I corrected the Markdown error (I had backwards URL syntax), and I made the text change. I'll include an embedded map for the Seattle post next week.

**Comment by drewda** [comment]
@stevevance 👍 👍

**Pull request merged**